scalahost/nsc/src/main/scala/scala/meta/internal/scalahost/ScalahostPipeline.scala

```diff
@@ -14,6 +14,7 @@ import scala.meta.internal.semantic.DatabaseOps
 import scala.meta.internal.semantic.{vfs => v}
 import scala.meta.internal.semantic.{schema => s}
 import scala.tools.nsc.doc.ScaladocGlobal
+import scala.meta.internal.semantic.schema.Message.Severity
```

Comment:
I've removed all unused imports in the build https://github.com/scalameta/scalameta/pull/1004